uvtools/dspec.py

```diff
@@ -105,12 +450,14 @@ def high_pass_fourier_filter(data, wgts, filter_size, real_delta, clean2d=False,
         add_clean_residual : bool, if True, adds the CLEAN residual within the CLEAN bounds
             in fourier space to the CLEAN model. Note that the residual actually returned is
             not the CLEAN residual, but the residual in input data space.
-        linear : bool,
-             use aipy.deconv.clean if linear == False
-             if True, perform linear delay filtering.
+        mode : string,
+             choose from ['clean','dayenu','dft_interp']
+             use aipy.deconv.clean if 'clean'
+             use 'dayenu' if 'dayenu'
+             if 'dft_interp', then interpolates flagged channels with DFT modes.
```

Comment:
does `dft_interp` belong in high_pass_fourier_filter? not sure if dft_interp is doing a high pass filter

Reply:
I think it technically could go in here since it's basically interpolating and subtracting foreground power just like `clean` but I'm also tempted to remove the all non-clean functionality from `high_pass_fourier_filter` just to clean it up. I think the function was originally intended for clean and I made the mistake of adding more and more arguments until things to really messy.